Gemfile

```diff
@@ -60,8 +60,8 @@ gem 'pg'
 # gem 'debugger', group: [:development, :test]
 
 
-gem "blacklight", '~> 5.12.1'
-gem "geoblacklight", '~> 0.11.0'
+gem "blacklight"
+gem "geoblacklight", '~> 0.12.1'
```

Comment:
Yep, we are also updating our blacklight version. Partly to make sure that GeoBlacklight in prod works for Blacklight 5.18 in anticipation of updating to 6.0x